appyters/dimensionality_reduction_visualization/dimensionality_reduction_visualization.ipynb

```diff
@@ -351,9 +440,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%appyter code_exec\n",
     "counter = 1\n",
     "\n",
+    "if need_leiden:\n",
+    "    leiden_visualization()\n",
```

Comment:
Just put the contents of this function here instead of using it or make the function pure (i.e. all variables it depends on in the arguments of the function)